### PR TITLE
Added linter with rules set and fixed problems

### PR DESCRIPTION
**Pull request by ghost**
Added linter to project. Included recommended rules for tslint and eslint and configured own set of rules

### CODE REVIEW

**Comment by OlegAtSpark** [comment]
@DanAtSpark, I've added some changes for eslint config. Please take a look at these. I'm going to merge this PR if you have no comments or questions.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+[{*.js, *.js.map}]
+indent_size=2
+
+[*.{json, yml, .eslintrc}]
```
Comment (ghost) [inline .editorconfig]: We have two absolutely same rules here and bellow

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    "ecmaVersion": 2019,
+    "sourceType": "module"
+  },
+  "globals": {
+    "Cypress": false,
+    "cy": false,
+    "before": false,
+    "beforeEach": false,
+    "expect": false
+  },
```
Comment (ghost) [inline .eslintrc:20]: Why we're setting false to every global?

**Review by ghost — COMMENTED** [review]
@OlegAtSpark Fix this please

File: .eslintrc (hunk 2 above)
Comment (OlegAtSpark) [reply .eslintrc:20]: `false` is equivalent for 'readonly'. I've pushed commit to change it to avoid misunderstanding

File: .editorconfig (hunk 1 above)
Comment (OlegAtSpark) [reply .editorconfig]: `.eslintrc` should not be here. Thanks for noticing!

**Review by OlegAtSpark — APPROVED** [review]

**Pull request merged**